examples/label_studio_annotation/README.md

```diff
@@ -65,26 +65,52 @@ example.
 You should install the relevant integrations:
 
 ```shell
-zenml integration install label_studio pytorch azure pillow
+zenml integration install pytorch azure pillow
```

Comment:
label studio example no longer requires label studio integration?

Reply:
It's mentioned above. also you can't install all of these at the same time with `zenml integration install`